ecommerce/ecommerce/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+
+class Product(models.Model):
+    name = models.CharField(max_length=255)
+    description = models.TextField()
+    price = models.DecimalField(max_digits=10, decimal_places=2)
+    image = models.ImageField(upload_to='products/')
```

Comment:
We can have a separate model for image and create many to one relationship with product. (one product many images)
something like:
```python
class Image(models.Model):
    product = models.ForeignKey(Product, on_delete=models.CASCADE)
    image = models.ImageField(upload_to='products/')
```